### ISSUE TITLE
Generated code of init task should ensure separation from existing code

### ISSUE THREAD

**Issue by bmuschko** [gogradle/gogradle#118]
When running the `init` task the generated code in `build.gradle` isn't clearly separated from existing code.

For example:

```
golang {
    packagePath = 'github.com/bmuschko/link-verifier'
}dependencies {
    golang {
        build name:'github.com/mvdan/xurls', version:'d315b61cf6727664f310fa87b3197e9faf2a8513', transitive:false
        test name:'github.com/stretchr/testify', version:'69483b4bd14f5845b5a1e55bca19e954e827f1d0', transitive:false
    }
}
```

The task should identify if a new line is needed and clearly separate the logic so the build file stays readable.

**Comment by blindpirate** [comment]
Thank you for your advice, it will be fixed in next release.

**Comment by blindpirate** [comment]
Fixed by `0.6.5`.
